prime-router/metadata/hl7_mapping/ORU_R01/base/observation-result.yml

```diff
@@ -120,8 +120,11 @@ elements:
     hl7Spec: [ '%{hl7OBXField}-7' ]
 
   - name: observation-abnormal-flag
-    value: [ '%resource.interpretation.coding.code' ]
-    hl7Spec: [ '%{hl7OBXField}-8' ]
+    resource: '%resource.interpretation'
+    constants:
+      versionId: '2.7'
+      cweFieldPath: '%{hl7OBXField}-8'
+    schema: ../../common/datatype/ce-coded-with-exceptions
 
```

Comment:
Simple report started sending us the interpretation code. CA was throwing validation errors if version and other fields were not populated. Seems like this should be used for all receivers.